docs/download-upload-clone/development-plan.md

```diff
@@ -0,0 +1,71 @@
+Each phase assumes that given version of exported feature is fully importable.
+
+# Release 1
+
+* zip file generation
+* zip file storage
```

Comment:
@Sikora00 exactly. It involves many parts/pieces to be generated/dumped, effectively making it time and resource consuming, not to mention it may be large enough to be hundred of MBs.